### PR TITLE
Serverless: Hypershift weekly

### PR DESCRIPTION
**Pull request by mgencur**
https://issues.redhat.com/browse/SRVCOM-2475

### CODE REVIEW

**Comment by mgencur** [comment]
/pj-rehearse periodic-ci-openshift-knative-serverless-operator-main-4.12-hypershift-e2e-hypershift-412-continuous

**Comment by mgencur** [comment]
/pj-rehearse periodic-ci-openshift-knative-serverless-operator-main-4.12-hypershift-e2e-hypershift-412-continuous

**Comment by mgencur** [comment]
/pj-rehearse periodic-ci-openshift-knative-serverless-operator-main-4.12-hypershift-e2e-hypershift-412-continuous

**Comment by mgencur** [comment]
TBD: Increase node count to 4 after https://github.com/openshift-knative/serverless-operator/pull/2092 is merged.

**Comment by mgencur** [comment]
/pj-rehearse periodic-ci-openshift-knative-serverless-operator-main-4.12-hypershift-e2e-hypershift-412-continuous

**Comment by mgencur** [comment]
/pj-rehearse periodic-ci-openshift-knative-serverless-operator-main-4.12-hypershift-e2e-hypershift-412-continuous

**Comment by mgencur** [comment]
/pj-rehearse periodic-ci-openshift-knative-serverless-operator-main-4.12-hypershift-e2e-hypershift-412-continuous

**Comment by mgencur** [comment]
/pj-rehearse periodic-ci-openshift-knative-serverless-operator-main-4.12-hypershift-e2e-hypershift-412-continuous

File: ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main-periodics.yaml [hunk 1]
```diff
@@ -1296,17 +1225,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-knative-serverless-operator-main-4.12-vsphere-e2e-vsphere-ocp-412-continuous
-  reporter_config:
```
Comment (cardil) [inline ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main-periodics.yaml:1299]: Why is this reporter config deleted everywhere?

**Review by cardil — COMMENTED** [review]
Looks good, apart from one quaestion:

File: ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main-periodics.yaml (hunk 1 above)
Comment (mgencur) [reply ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main-periodics.yaml]: Fixed.

**Pull request merged**